src/content/api/cli.md

```diff
@@ -141,6 +141,16 @@ Invocation                               | Resulting environment
 
 T> See the [environment variables](/guides/environment-variables) guide for more information on its usage.
 
+### Config Options
+
+Parameter                 | Explanation                                 | Input type | Default
+------------------------- | ------------------------------------------- | ---------- | ------------------
+`--config`                | Path to the config file                     | string     | webpack.config.js or webpackfile.js
+`--config-register, -r`   | Preload one or more modules before loading the webpack configuration | array | 
+`--config-name`           | Name of the config to use                   | string     | 
+`--env`                   | Environment passed to the config, when it is a function | 
+`--mode`                  | Mode to use, either "development" or "production" | string      | 
```

Comment:
isn't this `production` by default?

Reply:
nope, it throws if not supplied

Reply:
It shows a warning:
```
WARNING in configuration
The 'mode' option has not been set. Set 'mode' option to 'development' or 'production' to enable defaults for this environment.
```
but still bundle the app

Reply:
yeah, but it bundles the app as in `production` right? that's what I meant.

Reply:
https://github.com/webpack/webpack/blob/4f399322ad1922ae7bafdeea1c4d83bf9c9851a7/lib/WebpackOptionsDefaulter.js#L284

Reply:
It seems that is the case
![image](https://user-images.githubusercontent.com/1002461/37337196-d303471e-26b3-11e8-964c-9a54729d40a3.png)

Reply:
Correct, we default to production so people don't have consequences for not caring about the mode: property. However, we still will warn because we want users to be aware of being able to set the property.